Cython/Compiler/ModuleNode.py

```diff
@@ -2886,6 +2886,7 @@ def generate_module_cleanup_func(self, env, code):
 
         code.putln('static void %s(CYTHON_UNUSED PyObject *self) {' %
                    Naming.cleanup_cname)
+        code.putln("#if !CYTHON_COMPILING_IN_LIMITED_API")
         if Options.generate_cleanup_code >= 2:
```

Comment:
Wouldn't it still be necessary to generate the cleanup code, just targeting the references in the module state? Or are all of these references gone now?

Reply:
We do need to generate the cleanup code! I was crashing on cleaning up the dictionary but I was able to just guard that instead